### ISSUE TITLE
[LINT_BUG]: duplicate package imports aren't linted

### ISSUE THREAD

**Issue by TymekDev** [Appsilon/box.linters#149]
### box.linters version

v0.10.5

### Sample source code to lint

code <- "box::use(
  shiny,
  shiny,
)

shiny$shinyApp()"

lintr::lint(
  code,
  linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters)
)

### Lint command used

```r
code <- "box::use(
  shiny,
  shiny,
)

shiny$shinyApp()"

lintr::lint(
  code,
  linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters)
)
```

### Lint result

None, linter passes.

### Expected result

I expected to receive a lint telling me about a doubled import.
